packages/nodes-base/nodes/Gllue/services.ts

```diff
@@ -0,0 +1,28 @@
+import {IDataObject,} from 'n8n-workflow';
```

Comment:
I'm keeping this in services.ts to ease merge: I'll move the code into GenericFunctions.ts after merge is done